odoo/addons/openupgrade_records/lib/compare.py

```diff
@@ -325,21 +376,37 @@ def match(match_fields, match_type='direct'):
 
     sorted_records = sorted(
         old_records + new_records + moved_records + renamed_records +
-        modified_records,
+        modified_records + updated_records,
         key=lambda k: (k['model'], 'old' in k, k['name'])
     )
     for entry in sorted_records:
         content = ''
         if 'old' in entry:
-            content = 'DEL %(model)s: %(name)s' % entry
+            label = 'DEL'
+            if 'upd' in entry:
+                label += '(UPD)'
+            entry['label'] = label
+            content = '%(label)s %(model)s: %(name)s' % entry
             if 'moved' in entry:
-                content += ' [potentially moved to %(moved)s module]' % entry
+                potential = ''
+                if 'potent' in entry:
+                    potential = 'potentially '
+                entry['potent'] = potential
+                content += ' [%(potent)smoved to %(moved)s module]' % entry
             elif 'renamed' in entry:
                 content += ' [renamed to %(renamed)s module]' % entry
         elif 'new' in entry:
-            content = 'NEW %(model)s: %(name)s' % entry
+            label = 'NEW'
+            if 'upd' in entry:
+                label += '(UPD)'
```

Comment:
I see weird to see `NEW(UPD)`. Better just `UPD`:
```suggestion
                label = '(UPD)'
```

Reply:
And what about the DEL(UPD) cases?